unidist/core/backends/mpi/core/communication.py

```diff
@@ -129,6 +130,26 @@ def __init__(self, comm):
             self.topology[host][host_rank] = global_rank
             self.host_by_rank[global_rank] = host
 
+        mpi_hosts = MpiHosts.get()
+        if mpi_hosts is not None:
+            host_list = mpi_hosts.split(",") if mpi_hosts is not None else ["localhost"]
```

Comment:
Two identical checks `if mpi_hosts is not None`.

Reply:
Thanks. has been fixed